.eslintrc.cjs

```diff
@@ -2,38 +2,40 @@ module.exports = {
   root: true, // This fixes issue when running lint fix command: https://stackoverflow.com/questions/55060228/eslint-couldnt-find-the-plugin-eslint-plugin-typescript-eslint
   env: { browser: true, es2020: true },
   parser: '@typescript-eslint/parser',
-  parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
-  plugins: ['import', 'styled-components-a11y'],
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: { jsx: true },
+  },
+  plugins: ['import', '@typescript-eslint', 'styled-components-a11y'],
```

Comment:
why we need @typescript-eslint?

Reply:
It seems we don't need to specify this in both `plugins` and `extends`. So I removed it from plugins https://github.com/sendbird/chat-ai-widget/pull/180/commits/dccade2231ce2c82258e93fbf8fbdaae43e576ae